File: examples/docker-in-e2b/js/template/build.prod.ts
import { Template, defaultBuildLogger } from 'e2b'
import { template } from './template'

async function main() {
  await Template.build(template, {
    alias: 'e2b-with-docker',
    onBuildLogs: defaultBuildLogger(),
  });
}

main().catch(console.error);

File: examples/openai-codex-in-sandbox-python/build_prod.py
from e2b import Template, default_build_logger
from .template import template


if __name__ == "__main__":
    Template.build(
        template,
        alias="openai-codex",
        cpu_count=4,
        memory_mb=4096,
        on_build_logs=default_build_logger(),
    )
